geoengine/raster.py

```diff
@@ -17,6 +18,8 @@ class RasterTile2D:
     geo_transform: gety.GeoTransform
     crs: str
     time: gety.TimeInterval
+    band: int
+    tile_idx: Tuple[int, int]
```

Comment:
Actually, I don't see any place where you need the tile index.
Maybe remove (=do not add) it?

Reply:
I do use it when merging the tiles. I create a stacked x array for all the bands of a spatial tile and then combine the stacked arrays on the coordinates.

Reply:
lets see the discussion about the xarray API and then close or tackle this